### PR TITLE
Symfony5 support

### PR DESCRIPTION
**Pull request by tacman**
I've made some changes that allow this bundle to work with Symfony 5.  I'm not familiar with how to handle tests, travis, etc, though.

### CODE REVIEW

File: src/Command/PopulateCommand.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function populateIndex(OutputInterface $output, $index, $reset, $options)
     {
         $event = new IndexPopulateEvent($index, $reset, $options);
-        $this->dispatcher->dispatch(IndexPopulateEvent::PRE_INDEX_POPULATE, $event);
+        $this->dispatcher->dispatch($event, IndexPopulateEvent::PRE_INDEX_POPULATE);
```
Comment (XWB) [inline src/Command/PopulateCommand.php]: This won't work on < Symfony 4.3

File: src/Event/IndexEvent.php [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 namespace FOS\ElasticaBundle\Event;
 
-use Symfony\Component\EventDispatcher\Event;
+use Symfony\Contracts\EventDispatcher\Event;
```
Comment (XWB) [inline src/Event/IndexEvent.php]: Older Symfony versions might not have the contracts component.

File: src/Command/PopulateCommand.php (hunk 1 above)
Comment (tacman) [reply src/Command/PopulateCommand.php]: So how is this handled then?  A new version of the bundle?  Symfony 4.4 handles it by checking the argument order and calling a legacy handler (and throwing a deprecation warning).  Obviously, we don't want to not support Symfony 5 in favor of supporting older versions of Symfony, so how is this handled? (I don't have much experience with supporting bundles, even branches, tagging and pull requests are new territory for me).

Comment (tacman) [reply src/Command/PopulateCommand.php]: Um, so what's a better solution for supporting Symfony 5?  A LegacyDispatcher class?  How do you handle Contracts?  Is it something you can do, as this is a key bundle for some of us.  Thanks.

Comment (franmomu) [reply src/Command/PopulateCommand.php]: I would check if `$dispatcher` implements `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` [as suggested here](https://github.com/liip/LiipImagineBundle/pull/1212#issuecomment-523840038).

File: src/Event/IndexEvent.php (hunk 2 above)
Comment (franmomu) [reply src/Event/IndexEvent.php]: You could create [a `BCEvent` and extend from it](https://github.com/liip/LiipImagineBundle/pull/1218).

**Comment by franmomu** [comment]
I guess this should be done in https://github.com/FriendsOfSymfony/FOSElasticaBundle/tree/5.1.x instead of `master`.

**Comment by XWB** [comment]
https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1592

**Pull request closed without merging**